src/test/java/teammates/test/driver/TestProperties.java

```diff
@@ -137,6 +143,64 @@ public static String extractVersionNumber(String inputString) {
         return inputString.substring(startPos, endPos).replace("-", ".").trim();
     }
     
-    
+    public void verifyReadyForGodMode() {
+        if (!inst().isDevServer()) {
+            Assumption.fail("God mode regeneration works only in dev server.");
+        }
+        if (areTestAccountsDefaultValues()) {
+            Assumption.fail("Please change ALL the default accounts in test.properties in order to use God mode,"
+                            + "e.g change test.student1.account from alice.tmms to alice.tmms.example");
```

Comment:
Instead of simply asking them to change it, we can ask them to append their names. This way we dont need to handle so many special cases. What do you think?
We can make this a convention and avert the problem :-)